common-model/src/main/java/com/sequenceiq/common/api/cloudstorage/StorageLocationBase.java

```diff
@@ -1,24 +1,27 @@
 package com.sequenceiq.common.api.cloudstorage;
 
+import java.io.Serializable;
+
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.common.model.CloudStorageCdpService;
 
 import io.swagger.annotations.ApiModel;
 
 @ApiModel
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class StorageLocationBase {
+public class StorageLocationBase implements Serializable {
```

Comment:
why do you need Serializable?

Reply:
It's required by Spotbugs, there is a rule that every REST response should implement it.